spark/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -562,6 +562,125 @@ class DeltaTable private[tables](
           TableFeatureProtocolUtils.FEATURE_PROP_SUPPORTED))
     toDataset(sparkSession, alterTableCmd)
   }
+
+  /**
+   * Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
+   *
+   * @param target The path or table name to create the clone
+   * @param replace Whether to replace anything at the destination with the clone
+   * @param properties Any table properties to override in the clone
+   * @since 4.0.0
```

Comment:
```suggestion
   * @since 3.3.0
```

The next Delta Release is 3.3.0, which we should make it.

Reply:
Is there a planned 3.3.0 release? I thought the next release was going to be 4.0.0 on Spark 4.0

Reply:
@Kimahriman https://github.com/delta-io/delta/milestone/28

Reply:
I see, "tentative" so somebody will just come update this if that doesn't happen